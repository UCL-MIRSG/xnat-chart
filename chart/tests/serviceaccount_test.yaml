templates:
  - templates/serviceaccount.yaml

tests:
  - it: serviceaccount manifest should match snapshot
    set:
      serviceAccount.create: true
    asserts:
      - matchSnapshot:
          path: metadata.name
