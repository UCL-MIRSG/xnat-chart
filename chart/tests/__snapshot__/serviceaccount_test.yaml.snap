serviceaccount manifest should match snapshot:
  1: |
    RELEASE-NAME-xnat
